app/views/rescue_requests/show.html.erb

```diff
@@ -11,6 +11,14 @@
 <p class="text-info"><strong>Current status:</strong> <%= @request.request_status&.name || '(no status)' %></p>
 <p class="text-warning"><strong>Current medical status:</strong> <%= @request.medical_status&.name || '(no status)' %></p>
 
+<% if @duplicates.count > 0 %>
+<p><%= pluralize @duplicates.count, 'Duplicate' %>: <%= raw(@duplicates.map { |dupe| link_to sanitize(dupe.name), disaster_request_path(disaster_id: @disaster.id, num: dupe.id) }.join(", ")) %></p>
```

Comment:
`dupe.name` doesn't need sanitizing

Reply:
It's user inputted, so it does. Unless link_to does it?

Reply:
No, it doesn't. Rails does it. As a general rule, assume Rails handles sanitization for you, unless you're trying to render user-inputted HTML.

Reply:
Yeah, but the <a> tags get sanitized out if I do it any other way.

Reply:
Keep the `raw` call, yes, but you don't need the `sanitize` call here.